src/wizard/dotnet/awslambda.md

```diff
@@ -48,4 +48,24 @@ public class LambdaEntryPoint : Amazon.Lambda.AspNetCoreServer.APIGatewayProxyFu
 }
 ```
 
+### Verification
+
+You can verify your setup by throwing an exception from a function:
+
+```csharp
+[Route("api/[controller]")]
+public class BadController
+{
+    [HttpGet]
+    public string Get() => throw null;
+}
+``
+
+And make a request to that lambda:
+
+```sh
+curl -X GET -I https://url.of.server.aws/api/bad
+``
```

Comment:
You didn't add the extra ` I pointed out

Reply:
I thought I did, sorry if I missed it. Thanks for taking care of this.